src/js/ScrollArea.jsx

```diff
@@ -248,65 +253,106 @@ export default class ScrollArea extends React.Component{
 
         let newState = this.composeNewState(-deltaX, -deltaY);
 
-        if((newState.topPosition && this.state.topPosition !== newState.topPosition) ||
-           (newState.leftPosition && this.state.leftPosition !== newState.leftPosition)) {
+        if ((newState.topPosition && this.state.topPosition !== newState.topPosition) ||
+            (newState.leftPosition && this.state.leftPosition !== newState.leftPosition)) {
             e.preventDefault();
             e.stopPropagation();
         }
 
         this.setStateFromEvent(newState, eventTypes.wheel);
+        this.focusContent();
+    }
+
+    handleKeyDown(e) {
+        // only handle if scroll area is in focus
+        if (e.target.tagName.toLowerCase() !== 'input') {
+            let deltaY = 0;
+            let deltaX = 0;
+            let lineHeight = this.lineHeightPx ? this.lineHeightPx : 10;
+
+            switch (e.keyCode) {
+                case 33: // page up
+                    deltaY = lineHeight * 10;
```

Comment:
It's `page up` button so I think that it will be better to set delta to `this.state.containerHeight - lineHeight`